app/src/main/java/app/revanced/integrations/youtube/swipecontrols/SwipeControlsConfigurationProvider.kt

```diff
@@ -104,5 +104,17 @@ class SwipeControlsConfigurationProvider(
     val shouldSaveAndRestoreBrightness: Boolean
         get() = Settings.SWIPE_SAVE_AND_RESTORE_BRIGHTNESS.get()
 
+    /**
+     * should the auto-brightness be enabled at the lowest value of the brightness gesture
+     */
+    val shouldEnableLowestValueAutoBrightness: Boolean
+        get() = Settings.SWIPE_ENABLE_LOWEST_VALUE_AUTO_BRIGHTNESS.get()
+
+    /**
+     * variable that stores the brightness gesture value
+     */
+    var savedScreenBrightnessValue: Float
+        get() = Settings.SWIPE_BRIGHTNESS_VALUE.get()
+        set(value) = Settings.SWIPE_BRIGHTNESS_VALUE.save(value)
```

Comment:
In the future, this can be simplified by adding get set operators and delegating getter setter properties, not a matter of this PR though.